app/controllers/api/v1/members_controller.rb

```diff
@@ -0,0 +1,59 @@
+class Api::V1::MembersController < ApplicationController
+
+    include AuthenticationCheck
+
+    before_action :is_user_logged_in
+    before_action :set_member, only: [:show, :update, :destroy]
+  
+    # GET /members
+    def index
+      @members = Member.all
+      render json: @members
+    end
+  
+    # GET /members/:id
+    def show
+      # your code goes here
+     
+     # render json: @member
+      render json: { message: "If you see this, you're in!" }
```

Comment:
You might want to change it back for the next assignment.

```suggestion
      render json: @member
```